2024/opmodes/TwoStickTeleop.java

```diff
@@ -138,8 +147,8 @@ public void init() {
         // more investigation
         drivetask = new TeleopDriveTask(this, scheme, frontLeft, frontRight, backLeft, backRight);
 
-        liftMotorTask = new OneWheelDriveTask(this, liftMotor, true);
-        liftMotorTask.slowDown(false);
+        // liftMotorTask = new OneWheelDriveTask(this, liftMotor, true);
+        // liftMotorTask.slowDown(false);
```

Comment:
Please don't leave commented out software in pull requests.  It is fine to comment out software as you are experimenting and debugging, but software presented for merge should be free of this sort of commented out software.